src/rdkafka_conf.h

```diff
@@ -360,7 +360,8 @@ struct rd_kafka_conf_s {
         } eos;
 	int    queue_buffering_max_msgs;
 	int    queue_buffering_max_kbytes;
-	int    buffering_max_ms;
+        double buffering_max_ms_dbl; /**< This is the configured value */
```

Comment:
having both of these here is unfortunate since they're redundant. i don't see a good way around it, but if you do, worth changing.